src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/movements_out_on_given_day.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "offenderNo": "Z0020ZZ",
+    "dateOfBirth": "1966-01-01",
+    "reasonDescription": "Funerals And Deaths",
+    "timeOut": "00:00:00",
+    "firstName": "Burt",
+    "lastName": "Reynolds"
+  }
+]
```

Comment:
Would it be worth adding some data for the filtering? I saw that this file was used for both resource tests, which I assume is because the test database only has one move for the date. I know it can be a pain to add more data, so definitely not blocking.

Reply:
I nearly got away with it too!